src/style/content/conversation/message-list.less

```diff
@@ -307,8 +307,8 @@
   display: flex;
   max-width: @conversation-max-width;
   justify-content: space-between;
+  padding-top: 6px;
   padding-left: @conversation-message-sender-width;
-  padding-block: 4px 8px;
```

Comment:
The padding-bottom was apparently not part of the specs.
Design asked to align on 6px